modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1876,6 +1878,105 @@ public void testGetObjectEntriesAggregationFacets() throws Exception {
 		Assert.assertEquals(facetValue.getNumberOfOccurrences(), (Integer)2);
 	}
 
+	@Test
+	public void testGetRelatedObjectEntriesAccountEntryRestrictions()
```

Comment:
@pedro-oliveira446 rename to `testGetObjectEntryRelatedObjectEntriesAccountRestriction()` to follow the pattern define in this class.